src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -1396,40 +1396,40 @@ public sealed partial class GraphicsPath : System.MarshalByRefObject, System.ICl
-        public bool IsVisible(System.Drawing.Point pt, System.Drawing.Graphics graphics) { throw null; }
+        public bool IsVisible(System.Drawing.Point pt, System.Drawing.Graphics? graphics) { throw null; }
         public bool IsVisible(System.Drawing.PointF point) { throw null; }
-        public bool IsVisible(System.Drawing.PointF pt, System.Drawing.Graphics graphics) { throw null; }
+        public bool IsVisible(System.Drawing.PointF pt, System.Drawing.Graphics? graphics) { throw null; }
         public bool IsVisible(int x, int y) { throw null; }
-        public bool IsVisible(int x, int y, System.Drawing.Graphics graphics) { throw null; }
+        public bool IsVisible(int x, int y, System.Drawing.Graphics? graphics) { throw null; }
         public bool IsVisible(float x, float y) { throw null; }
-        public bool IsVisible(float x, float y, System.Drawing.Graphics graphics) { throw null; }
+        public bool IsVisible(float x, float y, System.Drawing.Graphics? graphics) { throw null; }
         public void Reset() { }
         public void Reverse() { }
         public void SetMarkers() { }
         public void StartFigure() { }
         public void Transform(System.Drawing.Drawing2D.Matrix matrix) { }
         public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect) { }
-        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix matrix) { }
-        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix matrix, System.Drawing.Drawing2D.WarpMode warpMode) { }
-        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix matrix, System.Drawing.Drawing2D.WarpMode warpMode, float flatness) { }
+        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix? matrix) { }
+        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix? matrix, System.Drawing.Drawing2D.WarpMode warpMode) { }
+        public void Warp(System.Drawing.PointF[] destPoints, System.Drawing.RectangleF srcRect, System.Drawing.Drawing2D.Matrix? matrix, System.Drawing.Drawing2D.WarpMode warpMode, float flatness) { }
         public void Widen(System.Drawing.Pen pen) { }
-        public void Widen(System.Drawing.Pen pen, System.Drawing.Drawing2D.Matrix matrix) { }
-        public void Widen(System.Drawing.Pen pen, System.Drawing.Drawing2D.Matrix matrix, float flatness) { }
+        public void Widen(System.Drawing.Pen pen, System.Drawing.Drawing2D.Matrix? matrix) { }
+        public void Widen(System.Drawing.Pen pen, System.Drawing.Drawing2D.Matrix? matrix, float flatness) { }
     }
     public sealed partial class GraphicsPathIterator : System.MarshalByRefObject, System.IDisposable
     {
```

Comment:
Below on row 1436, 1444, 1447 parameter `GraphicsPath path` allows null, but might not intended